src/api/records.tsx

```diff
@@ -186,7 +255,7 @@ const useRecordsPaginatedOptions = {
     }),
 };
 
-export const useRecordsPaginated = (): UseQueryResult<
+export const useRecordsPaginatedOld = (): UseQueryResult<
```

Comment:
Is this used anywhere? If not then we can delete it

Reply:
Good spot, leftover junk